Src/DependencyCollector/Shared/Implementation/DependencyCollectorEventSource.cs

```diff
@@ -266,9 +266,17 @@ private string GetApplicationName()
             string name;
             try
```

Comment:
We should probably cache the value of name and not make the reflection calls each time.